src/coreclr/vm/methodtable.h

```diff
@@ -577,6 +578,9 @@ class MethodTable
 
     LPCWSTR GetPathForErrorMessages();
 
+    // This method boxes a value type using a given method table pointer
```

Comment:
Also move this to comutilnative.h

Reply:
Done in 99afb976bbaeb229fb0e48f200a9e256c7562b78.